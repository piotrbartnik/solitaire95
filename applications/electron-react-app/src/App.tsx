import React from "react";
import { HelloWorld } from "my-module";

function App() {
  return (
    <>
      <HelloWorld />
    </>
  );
}

export default App;
