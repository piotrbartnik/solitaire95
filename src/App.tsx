import React, { ReactElement } from "react";
import styles from "./App.module.scss";
import card from "./card.png";

const App = (): ReactElement => {
  return (
    <div className={styles.App}>
      <img src={card} />
    </div>
  );
};

export default App;
