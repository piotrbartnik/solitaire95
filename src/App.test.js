import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('renders App page', () => {
  it('and checks basic snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot();
  })
});
