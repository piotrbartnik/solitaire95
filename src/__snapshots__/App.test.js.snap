// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App page and checks basic snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      src="card.png"
    />
  </div>
</DocumentFragment>
`;
